From feb5d8f18710919a8cfe76b0cdc705c541a24d96 Mon Sep 17 00:00:00 2001
From: Dmitri Epshtein <dima@marvell.com>
Date: Sat, 18 Jun 2016 20:37:40 +0300
Subject: [PATCH 507/538] pp3: Update PP3 driver version to NSS:16.06.1

Changes from prevoius version: NSS:16.06.0
- Add pp3 entry to dts files for all boards
- Fix interrupt enable for message channels
- Fix low performance issue
- Fix compilation warnings

Change-Id: Ib0572fd152c16b286713d79368c6b93539213129
Signed-off-by: Dmitri Epshtein <dima@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/30529
Reviewed-by: Dovrat Zifroni <dovrat@marvell.com>
Tested-by: Dovrat Zifroni <dovrat@marvell.com>
Reviewed-by: Yelena Krivosheev <yelena@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
---
 drivers/net/ethernet/marvell/pp3/platform/mv_pp3_version.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/pp3/platform/mv_pp3_version.c b/drivers/net/ethernet/marvell/pp3/platform/mv_pp3_version.c
index 07f83ab..02668a3 100644
--- a/drivers/net/ethernet/marvell/pp3/platform/mv_pp3_version.c
+++ b/drivers/net/ethernet/marvell/pp3/platform/mv_pp3_version.c
@@ -27,7 +27,7 @@ static struct mv_pp3_version mv_pp3_driver_version = {
 	.name = "NSS",
 	.major_x = 16,
 	.minor_y = 6,
-	.local_z = 0,
+	.local_z = 1,
 	.debug_d = 0
 };
 
-- 
1.9.1

